.travis.yml

```diff
@@ -1,10 +1,19 @@
 # borrowed from https://github.com/goreleaser/homebrew-tap
```

Comment:
This is the part I am least sure of - I'm not super familiar with travis. Ideally what we would do here long term is run some CI against any PR to make sure the chart isn't horribly deformed, and then on creation of a tag (release) in master, we would package+index, then push that package to the associated github release.